pkg/containerd/docker-bake.hcl

```diff
@@ -21,7 +21,7 @@ variable "CONTAINERD_REPO" {
 
 # Sets the containerd ref.
 variable "CONTAINERD_REF" {
-  default = "v1.6.14"
+  default = "v1.6.15"
```

Comment:
It's a bit confusing that we have a default both in the makefile and in the bake file 🤔

Reply:
Agree, since https://github.com/docker/buildx/pull/1449 we could just set `default: null`.

Also https://github.com/docker/packaging/pull/73 should be merged first as we should always build from default branch imo. Let me know what you think.